src/views/Recorder/Recorder.tsx

```diff
@@ -95,7 +97,7 @@ export function Recorder() {
     try {
       setRecorderState('saving')
 
-      if (proxyData.length === 0) {
+      if (proxyData.length === 0 && browserEvents.length === 0) {
```

Comment:
I did this while experimenting with the toggle for network capture, but I'm keeping it because it fixes another issue that I discovered: if you clear the request log and stop the recording, no recording will be created even if you have browser events.